environment_python_3_6.yml

```diff
@@ -1,37 +1,37 @@
 channels:
 - defaults
-- http://ssb.stsci.edu/astroconda
+- http://ssb.stsci.edu/astroconda-dev
```

Comment:
We shouldn't be relying on astroconda-dev.

Reply:
Yeah, I accidentally pasted changes from an old version 😬   But I've decided to ignore this PR anyways now that #549 is in, so this mistake won't take effect.